setup/docker.md

```diff
@@ -158,6 +158,40 @@ multi-machine scaling.
 
 For organizations, we recommend one Docker host per team of 5-10 developers.
 
+## Docker Compose
+
+   For a convenent way to run Coder in Docker, you can use [Docker Compose](https://docs.docker.com/compose/).
+
+   Simply create a new directory (for example, named `c4d`) and place the following
+   content in a file named `docker-compose.yml`:
+
+   ```yaml
+
+version: "3.5"
+services:
+  coder:
+    image: docker.io/codercom/coder:1.28.2
+    container_name: coderd
+    restart: unless-stopped
+    ports:
+      - 7080:7080/tcp
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ${HOME}/.coder:/var/run/coder
```

Comment:
Do you nee the `{ }`?

Reply:
It's just a habit of mine, it's probably not necessary.